packages/web-client/tests/acceptance/withdrawal-test.ts

```diff
@@ -294,6 +294,42 @@ module('Acceptance | withdrawal', function (hooks) {
       .containsText('You have successfully withdrawn tokens');
     assert.dom(epiloguePostableSel(1)).containsText('You withdrew');
     assert.dom(epiloguePostableSel(1)).containsText('You received');
+    assert
+      .dom(
+        '[data-test-withdrawal-transaction-confirmed-from] [data-test-bridge-item-network]'
+      )
+      .containsText('L2 test chain');
+    assert
+      .dom(
+        '[data-test-withdrawal-transaction-confirmed-from] [data-test-bridge-item-amount]'
+      )
+      .containsText('200.00 DAI.CPXD');
+    assert
+      .dom(
+        '[data-test-withdrawal-transaction-confirmed-from] [data-test-bridge-item-wallet]'
+      )
+      .containsText('0x1826...6E44');
+    assert
+      .dom(
+        '[data-test-withdrawal-transaction-confirmed-from] [data-test-bridge-item-depot]'
+      )
+      .containsText('0xB236...6666');
+    assert
+      .dom(
+        '[data-test-withdrawal-transaction-confirmed-to] [data-test-bridge-item-network]'
+      )
+      .containsText('L1 test chain');
+    assert
+      .dom(
+        '[data-test-withdrawal-transaction-confirmed-to] [data-test-bridge-item-amount]'
+      )
+      .containsText('200.00 DAI');
+    assert
+      .dom(
+        '[data-test-withdrawal-transaction-confirmed-to] [data-test-bridge-item-wallet]'
+      )
+      .containsText(layer1AccountAddress);
```

Comment:
I’m at EOD so I won’t do a thorough review today, but one comment would be that in discussions with L&M, the acceptance tests are starting to feel enormous, what do you think about moving these assertions into a component test? I took a step toward that [here](https://github.com/cardstack/cardstack/pull/1809/commits/37d07cd17705e83f39572aa34f8c8abe62d34214) for example, now the acceptance test for that card only has a single assertion. That card in particular doesn’t add anything to `workflowSession`, but I think it could be good to test that user actions result in values being stored under particular variables in the session state 🤔

I retain a desire for acceptance tests to assert on the flow of choices through the entire workflow, so maybe the user-entered `200` would be good to check for at least. Still figuring all this out! 💞